docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,124 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`DSS CLI is installed <install_DSS_CLI>` and :ref:`initialised <initialise_DSS>`.
+* Your machine has an Intel GPU.
+
+Install the Intel GPU plugin
+----------------------------
+
+To ensure DSS can utilise Intel GPUs:
+1. The Intel GPU drivers must be installed.
+2. MicroK8s must be set up to utilise the Intel GPU drivers.   
+
+Verify the Intel GPU drivers are installed in your machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To confirm that your machine has the Intel GPU drivers set up, install the `intel-gpu-tools` package:
+
+.. code-block:: bash
+
+   sudo apt install intel-gpu-tools
+
+To list the Intel GPU devices on your machine, run:
+
+.. code-block:: bash
+
+   intel_gpu_top -L
+
+If the drivers are correctly installed, you should receive information about your GPU device from the command's output.
```

Comment:
```suggestion
If the drivers are correctly installed, you should see information about your GPU device such as the following:
```
Can we add the output as a command bash code example?

Reply:
Just added an example output of an Intel discrete GPU